webapp/src/components/propertyValueElement.tsx

```diff
@@ -178,6 +182,32 @@ const PropertyValueElement = (props:Props): JSX.Element => {
                 readOnly={readOnly}
             />
         )
+    } else if (propertyTemplate.type === 'createdBy') {
+        return (
+            <UserProperty
+                value={card.createdBy}
+                readonly={true} // created by is an immutable property, so will always be readonly
+                onChange={() => {}} // since created by is immutable, we don't need to handle onChange
+            />
```

Comment:
If we are following the pattern of every property has it's own component, would it be too much proxy if createdBy had it's own component too which did just this? I'm not sure but it came to mind with what we had discussed before.

Reply:
Makes sense. Done.